client/src/components/SubstraitGraph.vue

```diff
@@ -4,7 +4,7 @@
     <div id="svgContainer">
       <div style="margin-right: 10vh; text-align: right">
         <button
-          ref="download_json"
+          id="copy-button"
```

Comment:
This one looks like it should still be download.

Reply:
corrected that, thanks!